### PR TITLE
Dealer of the Damned (Formerly-ish Six-Shot Sharper)

### PR DESCRIPTION
**Pull request by X3ros**
## About The Pull Request
This adds Six-Shot Sharper, a TETH abnormality, as well as its EGO, Luck of the Draw and its unique weapon, Death Dealer.

## Why It's Good For The Game
The idea behind the abnormality has been considered by a decent chunk of players to be potentially fun to have, and it has been approved by the lead dev, Kitsunemitsu.

## Changelog
:cl:
imageadd: Added Six-Shot Sharper and its EGO's sprites, courtesy of sponge_4781 on the LC13 Discord
code: Added Six-Shot Sharper and its associated EGO (Luck of the Draw and Death Dealer)
:cl:

## Shit that needs to be done
NOTE: For all intents and purposes, the abnormality and its EGO are completely functional and able to be put into the game. 
All that I'd like to change are some minor tweaks, such as making Death Dealer's sound when firing match the projectile it's using and re-adding a functional version of the dud round. Those can come later, though.

### CODE REVIEW

**Comment by X3ros** [comment]
MASSIVE thanks to Kirie and the other devs who were there for the "Learn to Code" events on the LC13 discord for helping bring this to fruition. Genuinely never could have done this without you guys.

File: code/datums/abnormality/_ego_datum/teth.dm [hunk 1]
```diff
@@ -330,3 +330,12 @@
...
 	cost = 20
+
+//Dealer of the Damned - Luck of the Draw
+/datum/ego_datum/weapon/luckdraw
+	item_path = /obj/item/gun/ego_gun/luckdraw
+	cost = 20
+
+/datum/ego_datum/armor/luckdraw
+	item_path = /obj/item/clothing/suit/armor/ego_gear/teth/luckdraw
+	cost = 20
```
Comment (InsightfulParasite) [inline code/datums/abnormality/_ego_datum/teth.dm:341]: Dont forget to leave a extra line of blank code at the end of files. Its a weird quirk of byond code that a extra blank line is needed in each file.

**Comment by Kitsunemitsu** [comment]
I'm willing to give this a test if it gets a new sprite.

Talk to our spriters about it.

**Comment by X3ros** [comment]
Devsprites replaced, many thanks to sponge_4781 on the LC13 discord for making these beautiful sprites.

**Comment by Kitsunemitsu** [comment]
Linter errors are a little fucky, just add some new lines at the end of everything.
I'm putting on TM.

**Review by Kitsunemitsu — APPROVED** [review]
Tested on the live server. People love it.
Merging.

**Pull request merged**